arch/arm/src/stm32f7/stm32_can.c

```diff
@@ -589,7 +596,7 @@ static void stm32can_reset(FAR struct can_dev_s *dev)
   uint32_t regbit = 0;
   irqstate_t flags;
 
-  caninfo("CAN%d\n", priv->port);
+  caninfo("CAN%" PRId8 "\n", priv->port);
```

Comment:
```suggestion
  caninfo("CAN%" PRIu8 "\n", priv->port);
```
`priv->port` has type `uint8_t`.

Reply:
There are other similar occurrences in this file.